packages/create-pantheon-decoupled-kit/__tests__/runESLint.test.ts

```diff
@@ -12,15 +14,48 @@ vi.mock('which-pm-runs');
 const config: CustomActionConfig<'runLint'> = {
 	type: 'runLint',
 };
-const outDir = `${process.cwd()}/__tests__/fixtures/runESLint`;
-const answers: ParsedArgs = { _: [], outDir };
+const outDir = (dir: 'withLint' | 'withoutLint') =>
+	`${process.cwd()}/__tests__/fixtures/runESLint/${dir}`;
 const plop = await nodePlop.default();
 
 describe('runEsLint()', () => {
-	afterEach(() => {
+	beforeEach(async () => {
+		await fs.ensureDir(path.resolve(outDir('withLint')));
+		await fs.ensureDir(path.resolve(outDir('withoutLint')));
+		fs.writeFileSync(
+			`${outDir('withLint')}/package.json`,
+			JSON.stringify({
+				scripts: {
+					lint: 'next lint',
+				},
+			}),
+		);
+		fs.writeFileSync(
+			`${outDir('withoutLint')}/package.json`,
+			JSON.stringify({
+				scripts: {},
+			}),
+		);
+	});
+	afterEach(async () => {
 		vi.resetAllMocks();
+		await fs.ensureDir(path.resolve(outDir('withLint')));
+		await fs.ensureDir(path.resolve(outDir('withoutLint')));
+
+		['package.json', 'node_modules'].forEach((path) => {
+			fs.rmSync(`${outDir('withLint')}/${path}`, {
+				force: true,
+				recursive: true,
+			});
+			fs.rmSync(`${outDir('withoutLint')}/${path}`, {
+				force: true,
+				recursive: true,
+			});
+		});
```

Comment:
Not sure we need this pattern here since the tests aren't manipulating the files in any way like the other tests are. I think you could just commit the fixtures instead of programmatically creating the files before/after.

Reply:
@CobyPear I ended up going with this pattern because committing `package.json` files to the fixtures proved to be troublesome. Running `pnpm test` with the committed files fails and requires `node_modules` to be installed at the file path. After installing `node_modules`,  `pnpm test` continues to fail since it then treats those folders as their own projects which has missing test coverage. Below is the error output. LMK if there is a better solution but I found programmatically creating the files before/after to be an easy workaround.

<img width="1119" alt="Screen Shot 2023-02-07 at 2 25 36 PM" src="https://user-images.githubusercontent.com/39682014/217380631-c1f9071c-8be9-4f9f-83af-d98923e6676a.png">

Reply:
I did not have the same error, I was able to run tests with a `package.json` inside the fixtures.
I do see them being picked up by the monorepo workspace, but we can add another filter in the `pnpm-workspace.yml` to prevent this:
`  - '!packages/create-pantheon-decoupled-kit/__tests__/**'`

Reply:
looking a bit closer at your screenshot, looks like you may be in a subdir and `vitest` isn't finding any tests.